docs/DRONECAN.md

```diff
@@ -0,0 +1,37 @@
+# mLRS Documentation: DroneCAN for ArduPilot Systems #
+
+([back to main page](../README.md))
+
+DroneCAN can be utilized for MAVLink and RC with mLRS instead of traditional serial communication if both the flight controller and receiver have the hardware necessary to support it.
+
+## Wiring
+
+TBD
+
+## ArduPilot Settings
+
+The following parameters need to be set to enable MAVLink and RC over DroneCAN.
+
+Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+
+- CAN_P1_DRIVER = 1
+- CAN_D1_PROTOCOL = 1
+- CAN_D1_UC_SER_EN = 1
+
+Configure the DroneCAN virtual serial port:
+
+- CAN_D1_UC_S1_BD = 57600
```

Comment:
the baudrate is actually irrelevant

Reply:
Want to remove then?

Reply:
have no opinion, just wanted to mention :)